compute_tools/src/extension_server.rs

```diff
@@ -174,6 +174,20 @@ pub async fn get_available_libraries(
         );
     }
 
+    // TODO: shouldn't we filter this by .so?
+    /*
+    right now we are getting weird stuff like
+    "pg_isolation_regress": RemotePath("v14/lib/pgxs/src/test/isolation/pg_isolation_regress"),
+    Makefile.shlib": RemotePath("v14/lib/pgxs/src/Makefile.shlib")
+    which we should not try to download...
```

Comment:
Where are we getting them from? We shouldn't have uploaded such files in a first place.

It shouldn't be too bad, because we always search by libname or extname,
but It's better to clean this up.

Reply:
I cleaned it up https://github.com/neondatabase/neon/pull/4606